docs/package.json

```diff
@@ -5,7 +5,8 @@
   "author": "Material-UI Team",
   "license": "MIT",
   "scripts": {
-    "build": "rimraf .next && cross-env NODE_ENV=production next build",
+    "build": "cross-env NODE_ENV=production next build",
```

Comment:
Cleaning before building was introduce in #13987 by @oliviertassinari. It seems like it was used for debugging which is why I put it in a dedicated task. Since nextjs actually recommends using build caches we should follow this by default.